etl/scripts-ccao-data-warehouse-us-east-1/ccao/ccao-legacy.R

```diff
@@ -262,3 +262,88 @@ cc_pifdb_piexemptre_dise %>%
     hive_style = TRUE,
     compression = "zstd"
   )
+
+
+##### CC_PIFDB_PIEXEMPTRE_OWNR #####
+files_cc_pifdb_piexemptre_ownr <- aws.s3::get_bucket_df(
+  bucket = AWS_S3_RAW_BUCKET,
+  prefix = "ccao/legacy/CC_PIFDB_PIEXEMPTRE_OWNR",
+  max = Inf
+) %>%
+  filter(Size > 0)
```

Comment:
All of the code in this file is very similar to the logic to extract the other exemptions that Dan established in https://github.com/ccao-data/data-architecture/pull/589. The only bit that is specific to the homeowner data are the column positions and types that we define below, along with the transformations we apply to those columns.